### PR TITLE
Attempt to fix client -> server backpressure for real

### PR DESCRIPTION
**Pull request by loganzartman**
This is an attempt to fix ut-amrl/robofleet#6, which is the issue that we are still seeing lag (a large time delay with messages queuing up) in webviz visualization when network bandwidth is limited.

This works using the Websocket ping/pong protocol, which allows for sending and echoing a payload. I assume that the Websocket server is handling all messages in a FIFO fashion, and I can therefore ping the server with a message ID and wait for it to pong back the same ID before sending more messages.

This fixes the issue reproduced using artificial bandwidth limiting that I described on the linked issue.

### CODE REVIEW

File: src/config.example.hpp [hunk 1]
```diff
@@ -80,10 +95,13 @@ static void configure_msg_types(RosClientNode& cn) {
...
   cn.register_local_msg_type<sensor_msgs::CompressedImage>(
-      "/stereo/left/image_raw/compressed", webviz_constants::left_image_topic, 10);
+      "/stereo/left/image_raw/compressed",
+      webviz_constants::left_image_topic,
+      10);
   cn.register_local_msg_type<sensor_msgs::CompressedImage>(
       "/stereo/right/image_raw/compressed",
-      webviz_constants::right_image_topic, 10);
+      webviz_constants::right_image_topic,
+      10);
```
Comment (loganzartman) [inline src/config.example.hpp:104]: these are just from running `make format`, which I should create git hooks for or something

**Review by kvsikand — COMMENTED** [review]
The ping pong method is waht I found as well when looking into this, looks good to me!

However, I don't see a diff for the server to handle the ping messages and send back pongs (unless this is magically built in to WS or something?)...lets land the server side of this first, if it exists? Since that way there will never be a time when the clients are broken

**Comment by loganzartman** [comment]
> The ping pong method is waht I found as well when looking into this, looks good to me!
> 
> However, I don't see a diff for the server to handle the ping messages and send back pongs (unless this is magically built in to WS or something?)...lets land the server side of this first, if it exists? Since that way there will never be a time when the clients are broken

**Review by kvsikand — APPROVED** [review]
:+1:

**Pull request merged**